src/picknik_ur_multi_arm_config/config/config.yaml

```diff
@@ -178,6 +178,7 @@ ros2_control:
     - "second_servo_controller"
     - "third_servo_controller"
     - "fourth_servo_controller"
+    - "servo_controller"
```

Comment:
hard coded topic name comes from this controller for joint jog (sliders)

Reply:
I think I need to get rid of the *_servo controllers because they aren't used anyways